### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Harkirat155**
Completion of solution code.

- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: guide/english/certifications/apis-and-microservices/basic-node-and-express/get-data-from-post-requests/index.md [hunk 1]
```diff
@@ -10,12 +10,22 @@ Just like using req.query we can do req.body to get our data. This challenge is
 
 In order to get data from a post request a general format is:
 
+##Hint
```
Comment (thecodingaviator) [inline guide/english/certifications/apis-and-microservices/basic-node-and-express/get-data-from-post-requests/index.md]:
```suggestion
## Hint
```

File: guide/english/certifications/apis-and-microservices/basic-node-and-express/get-data-from-post-requests/index.md [hunk 2]
```diff
@@ -10,12 +10,22 @@ Just like using req.query we can do req.body to get our data. This challenge is
...
 ```javascript
 app.post(PATH, function(req, res) {
   // Handle the data in the request
 });
 ```
 
+##Solution
```
Comment (thecodingaviator) [inline guide/english/certifications/apis-and-microservices/basic-node-and-express/get-data-from-post-requests/index.md]:
```suggestion
## Solution
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
There should be a space to demarcate the heading content and type

**Comment by RandellDawson** [comment]
Congrats @Harkirat155 on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**